tests/integration/mounts_sshfs.bats

```diff
@@ -28,13 +30,21 @@ function teardown() {
 	teardown_bundle
 }
 
-@test "runc run [rw bind mount of a ro fuse sshfs mount]" {
+@test "runc run [rw,dev,exec,suid bind mount of a ro,nodev,nosuid,noexec fuse sshfs mount]" {
+	# The "sync" option is used to trigger a remount with the below options.
+	# It serves no further purpose. Otherwise only a bind mount without
+	# applying the below options will be done.
 	update_config '	  .mounts += [{
 					type: "bind",
 					source: "'"$DIR"'",
 					destination: "/mnt",
-					options: ["rw", "rprivate", "nosuid", "nodev", "rbind"]
+					options: ["rw", "dev", "suid", "exec", "rprivate", "rbind", "sync"]
 				}]'
+	if [ $EUID -eq 0 ]; then
+		update_config '.linux.namespaces += [{type: "user"}]
+				| .linux.uidMappings += [{"hostID": 100000, "containerID": 0, "size": 65534}]
+				| .linux.gidMappings += [{"hostID": 100000, "containerID": 0, "size": 65534}]'
+	fi
 
```

Comment:
The original test has to be kept

Reply:
> > @kolyshkin: Like now?
> 
> Yes, thanks!
> 
> If you can rebase one more time, CI might even become green again :)

One more rebase done. After the rebase is before the next one :-)

> 
> Also, a nit in the first commit message: you have "roset" without a space, and an extra space before period.

Fixed commit message.